pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -2094,8 +2057,9 @@ public JSONObject rebalanceTable(final String rawTableName, TableType tableType,
           RebalanceSegmentStrategyFactory.getInstance().getRebalanceSegmentsStrategy(tableConfig);
       PartitionAssignment newPartitionAssignment =
           rebalanceSegmentsStrategy.rebalancePartitionAssignment(idealState, tableConfig, rebalanceUserConfig);
-      IdealState newIdealState = rebalanceSegmentsStrategy.rebalanceIdealState(idealState, tableConfig,
-          rebalanceUserConfig, newPartitionAssignment);
+      IdealState newIdealState =
+          rebalanceSegmentsStrategy.rebalanceIdealState(idealState, tableConfig, rebalanceUserConfig,
```

Comment:
This file has a LOT of formatting changes mixed in with actual code changes; this increases the review burden and can lead to review misses. Can you please set the editor to not reformat lines that are not updated? Please keep formatting only changes in a PR different from actual code changes.